src/components/crew/crew-view/crew-view.component.js

```diff
@@ -73,8 +75,8 @@ class CrewViewPage extends Component {
             notes: dataHelper.getNotes(licenseNumber),
             captainName: dataHelper.getCaptainName(licenseNumber),
             crewName: isCaptain ?
-              dataHelper.getCaptainName(licenseNumber):
-              dataHelper.getCrewName(licenseNumber),
+              dataHelper.getCaptainName(licenseNumber || captainName):
+              dataHelper.getCrewName(licenseNumber || crewName),
```

Comment:
The `dataHelper` functions find data using name when licence number is absent.

https://github.com/WildAid/o-fish-web/blob/14fbb743b9061e79ac4a48c8b9eb63a9194f66c6/src/components/partials/boarding-data.helper.js#L205-L213